src/lib/holocene/code-block.svelte

```diff
@@ -191,7 +191,7 @@
     <CopyButton
       {copyIconTitle}
       {copySuccessIconTitle}
-      class="absolute right-1 top-1 text-white hover:bg-slate-800"
+      class="absolute right-1 top-1 text-slate-300 hover:bg-slate-900 focus-visible:bg-slate-900 active:bg-slate-800"
```

Comment:
Maybe there's a way we can just enforce `dark` styles here (for `surface-interactive-secondary`) in both `light` and `dark` mode instead of just using the same text and bg colors?

Reply:
Updated to use `data-theme="dark"` [here](https://github.com/temporalio/ui/pull/2303/commits/c38aa12ce10fd69cc686b4689dffbbd418c6b33d).